modules/machine-autoscaler-cr.adoc

```diff
@@ -24,9 +24,11 @@ spec:
     kind: MachineSet <5>
     name: worker-us-east-1a <6>
 ----
-<1> Specify the machine autoscaler name. To make it easier to identify which machine set this machine autoscaler scales, specify or include the name of the machine set to scale. The machine set name takes the following form: `<clusterid>-<machineset>-<aws-region-az>`
-<2> Specify the minimum number machines of the specified type that must remain in the specified zone after the cluster autoscaler initiates cluster scaling. If running in AWS, GCP, Azure, or {rh-openstack}, this value can be set to `0`. For other providers, do not set this value to `0`.
+<1> Specify the machine autoscaler name. To make it easier to identify which machine set this machine autoscaler scales, specify or include the name of the machine set to scale. The machine set name takes the following form: `<clusterid>-<machineset>-<aws-region-az>`.
+<2> Specify the minimum number machines of the specified type that must remain in the specified zone after the cluster autoscaler initiates cluster scaling. If running in AWS, GCP, Azure, {rh-openstack}, or vSphere, this value can be set to `0`. For other providers, do not set this value to `0`.
 <3> Specify the maximum number machines of the specified type that the cluster autoscaler can deploy in the specified AWS zone after it initiates cluster scaling. Ensure that the `maxNodesTotal` value in the `ClusterAutoscaler` resource definition is large enough to allow the machine autoscaler to deploy this number of machines.
 <4> In this section, provide values that describe the existing machine set to scale.
 <5> The `kind` parameter value is always `MachineSet`.
 <6> The `name` value must match the name of an existing machine set, as shown in the `metadata.name` parameter value.
+
+If your machines are running in AWS, GCP, Azure, {rh-openstack}, or vSphere, you can set the `spec.minReplicas` value to `0`. You can save on costs by setting this value to zero for use cases such as running expensive or limited-usage hardware used for specialized workloads, or scaling a machine set with extra large machines. The machine autoscaler scales the machine set down to zero when the machines are not in use.
```

Comment:
i like this addition, the only issue i have with this is that a reader might get the impression they could set all their MachineSets to 0 and save some money, but this will cause issues in IPI deployments if they scale down the initial MachineSets for the compute plane. i'm not sure the best way to describe this, but i think it could create some confusion or at least a misunderstanding to a reader looking for optimizations.

Reply:
Good point! I can add a bit more context to not scale those initial machine sets for the compute plane down to 0. 👍

Reply:
yeah, i think that would be perfect

Reply:
@elmiko I added a note about not setting to 0 for the compute plane machine sets if you can PTAL. Thank you.